lua/octo/writers.lua

```diff
@@ -1,6 +1,7 @@
 local hl = require "octo.highlights"
```

Comment:
I think this is no longer used and can be removed

Reply:
Ups. I really need to get my Lua language server running to get such diagnostics. Stupid lazy developer. :laughing:

Reply:
:heavy_check_mark: